system.py

```diff
@@ -159,6 +162,15 @@ def checkTools(cls):
     ret = cls.checkVSDebugTools()
     
     return ret
+  
+  @classmethod
+  def checkPythonVersion(cls):
+
+    python_version_number = str(sys.version_info.major) + '.' + str(sys.version_info.minor) + '.' + str(sys.version_info.micro)
+    if sys.version_info.major is not 2:
```

Comment:
If you already added SUPPORTED_PYTHON_VERSION variable, check major number against it's major number. Don't use hardcoded value